scripts/dts/extract/globals.py

```diff
@@ -331,6 +331,52 @@ def build_cell_array(prop_array):
 
     return ret_array
 
+def child_to_parent_unmap(cell_parent, gpio_index):
```

Comment:
"Unmap"?

Reply:
I felt child_to_parent was not enough specific.
extract?

Reply:
`gpio_map_index_to_node` or something maybe (wonder if `index` could be `id` too).

Maybe `cell_parent` could just be called `node_path`, since it's not important to the functionality that it's a parent of something that you're passing in.

Reply:
`gpio_map_child_to_parent` ?